src/log.rs

```diff
@@ -0,0 +1,8 @@
+macro_rules! logc {
```

Comment:
It is a bit repetitive, but I think it might be nicer if you simply re-exported all of the log methods (error, warn, info, debug, trace) in this file. If you did that, then when you go to consume it, you wouldn't have to import both crate::log::logc in addition to log::level (behind a feature gate). You could just import logc and use its corresponding macro methods.

Reply:
Sounds good, I'll make that change.